cmd/helmExecute.go

```diff
@@ -96,19 +96,21 @@ func runHelmExecute(config helmExecuteOptions, helmExecutor kubernetes.HelmExecu
 			return fmt.Errorf("failed to execute helm dependency: %v", err)
 		}
 	case "publish":
-		if err := helmExecutor.RunHelmPublish(); err != nil {
+		targetURL, err := helmExecutor.RunHelmPublish()
+		if err != nil {
 			return fmt.Errorf("failed to execute helm publish: %v", err)
 		}
+		commonPipelineEnvironment.custom.remoteHelmChartPath = targetURL
```

Comment:
Isn't that just another missed opportunity to align artefact metadata in a standardized artifact struct, @OliverNocon & @anilkeshav27 & @CCFenner?